app/views/taxonomy_signups/confirm.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :title, @taxon['title'] %>
+
+<%= render 'govuk_component/breadcrumbs', breadcrumbs: @breadcrumbs %>
+
+<div class='email-signup column-two-thirds'>
+<div class="govuk-title length-long">
```

Comment:
This might work with http://govuk-component-guide.herokuapp.com/components/title